Tests/QuranAudioKitTests/DownloadedRecitersServiceTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  DownloadedRecitersServiceTests.swift
+//  
+//
+//  Created by Zubair Khan on 1/2/23.
+//
+
+import Foundation
+@testable import QuranAudioKit
+import XCTest
+
+class DownloadedRecitersServiceTests: XCTestCase {
+    private var service: DownloadedRecitersService!
+
+    override func setUpWithError() throws {
+        service = DownloadedRecitersService()
+    }
+
+    func testFilterDownloadedReciters() {
+        let reciter1 = createReciter(id: 1)
+        let reciter2 = createReciter(id: 2)
+        let reciter3 = createReciter(id: 3)
+        let reciter4 = createReciter(id: 4)
+        let reciter5 = createReciter(id: 5)
+
+        let path2 = URL(string: "file:///audio_files/dir2/")!
+        let path3 = URL(string: "file:///audio_files/dir3/")!
+        let path4 = URL(string: "file:///audio_files/dir4/")!
+        let path5 = URL(string: "file:///audio_files/dir5/")!
+
+        let allReciters: [Reciter] = Array(arrayLiteral: reciter1, reciter2, reciter3, reciter4, reciter5)
+        let downloadedRecitersPaths: [URL] = Array(arrayLiteral: path2, path3, path4, path5)
+
+        var recentReciters: [Reciter] = [] // test without any filtering of recent
+        var downloadedReciters = service.filterDownloadedReciters(allReciters, recentReciters, downloadedRecitersPaths)
+
+        XCTAssertEqual(downloadedReciters, [reciter2, reciter3, reciter4, reciter5])
```

Comment:
I'm not sure I understand how these specific reciters have been downloaded, I don't see anything in the setup method.